kayenta-core/src/main/java/com/netflix/kayenta/security/MetricAccountCredentials.java

```diff
@@ -0,0 +1,9 @@
+package com.netflix.kayenta.security;
+
+import java.util.List;
+
+public interface MetricAccountCredentials<T> extends AccountCredentials<T> {
```

Comment:
I don't see this interface used anywhere (nor do I see where it would be, so I think it extraneous).

Reply:
Ooops.